src/utils/platformServices.js

```diff
@@ -1,9 +1,22 @@
+import { useQuery } from 'react-query';
+
+import useChrome from '@redhat-cloud-services/frontend-components/useChrome';
+
 const authenticateUser = () => {
-  try {
-    return window.insights.chrome.auth.getUser();
-  } catch (e) {
-    throw new Error(`Error authenticating user: ${e.message}`);
-  }
+  console.warn('DO NOT USE');
+  return;
+};
+
+const useAuthenticateUser = () => {
+  const chrome = useChrome();
+
+  return useQuery('authenticated-user', () => {
+    try {
+      return chrome?.auth?.getUser();
```

Comment:
the same suggestion here:
```suggestion
      return chrome?.auth?.getUser?.();
```